Upgrade docs pre v1.0.0 (#1852)
solo-io/gloo-mesh: docs/content/operations/upgrading.md

description: Best practices for upgrading Gloo Mesh
---

{{< notice warning >}}

> This upgrade process is for Gloo Mesh, from a pre-1.0.0 state up to v1.0.0.

Reviewer: Does it also apply to Enterprise or just the OSS?
```suggestion
This upgrade process is for Gloo Mesh versions less than v1.0.0 up to v1.0.0.
```

Author: Just the OSS - the enterprise version did not have any upgrade docs that I'm aware of.

> This upgrade process is for Gloo Mesh versions less than v1.0.0 up to v1.0.0.

Author: ```suggestion
This upgrade process is for Gloo Mesh versions less than v1.0.0 up to, but not including, v1.0.0.
```

{{< /notice >}}

> Gloo Mesh is under active development. There may be breaking API changes with every minor version bump.

Reviewer 2: This is no longer true. Post 1.0.0 we are committed to not having breaking API changes between minor versions!

```suggestion
Gloo Mesh is under active development.
```
